### ISSUE TITLE
MSW Fullscreen does not use monitors native resolution

### ISSUE THREAD

**Issue by azjata** [cinder/Cinder#1391]
On MSW fullscreen does not make the window to turn to native resolution.
The only way to force the window to be full hd is to make a borderless window with 0,0 position and fixed size. In fullscreen mode 1920x1080 becomes 1536x864.
cinder_0.9.0_vc2013 & nvidia gpu

**Comment by richardeakin** [comment]
Do you have text scaling on in your display settings? It's on by default on win 8.

**Comment by paulhoux** [comment]
See also: https://forum.libcinder.org/topic/windos-8-1-disable-display-scaling#23286000002431065

**Comment by azjata** [comment]
Thanks!
Disabling text scaling in display settings did resolve the issue.
Would be great to have that mentioned somewhere in the docs in the setFullScreen for example.

**Comment by richardeakin** [comment]
Also it'd be great if we figured out a way so that cinder apps weren't affected by the scaling, @paulhoux  do you know if we could we solve it in the project properties or do we need that bit of extra code too (`SetProcessDpiAwareness()`)?

Going to reopen as I think its an issue we should try to look into..

**Comment by paulhoux** [comment]
Yeah, just enable DPI awareness by default and we should be good. It's a project setting.

**Comment by paulhoux** [comment]
This should be added to TinderBox, actually. Add DPI awareness to the project's manifest. Can be done by adding a few lines to the ```vcproj``` file.
